index.js

```diff
@@ -120,8 +120,13 @@ ENS.prototype.resolver = function(name) {
     }
 
     var abi = resolverInterface;
-    if((callback && arguments.length == 4) || (!callback && arguments.length == 3))
-        resolverInterface = arguments[arguments.length - 2];
+
+    var params = {};
```

Comment:
It doesn't look like params is used anywhere? Did you mean to say `abi` instead?